docker/compose/nginx/files/nginx.conf

```diff
@@ -59,43 +59,8 @@ http {
     listen              *:9090;
     access_log          /dev/null main;
 
-    location ~ (/app/kibana|/app/sense|/bundles/|/kibana4|/status|/plugins|/elasticsearch|/api/sense/proxy|/kibana|/kibana/goto) {
```

Comment:
I suppose all of this is not needed for docker, but does sense still work if we remove the path to it?

Reply:
Sense is no longer present in kibana, they removed the plugin entirely. They offer the same functionality built into kibana.

Reply:
oh nice! that's great news :)